pkg/agent/datamodel/types.go

```diff
@@ -122,6 +122,14 @@ const (
 	WasmWasi WorkloadRuntime = "WasmWasi"
 )
 
+// OutboundType describes the options for outbound internet access.
+type OutboundType string
+
+const (
+	OutboundTypeNone  string = "none"
```

Comment:
you should use `OutboundType` as the type here since you define it right above

OutboundType == string, though string != OutboundType

Reply:
Right - since 

nbc.OutboundType is going to receive a string - it might be better to get rid of the OutboundType all together and just use strings - since I think the check would have to look like 

strings.EqualFold(nbc.OutboundType, string(datamodel.OutboundTypeBlock)) || strings.EqualFold(nbc.OutboundType, string(datamodel.OutboundTypeNone))